src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/files/jupyterlab/overrides.json

```diff
@@ -50,13 +51,8 @@
                 "rank": 1000,
                 "items": [
                     {
-                        "command": "help:open",
-                        "rank": 1,
-                        "args": {
-                            "url": "/conda-store",
-                            "text": "Environment Management",
-                            "newBrowserTab": true
```

Comment:
Without this, does the extension still opens as a separate tab?

Reply:
That code was opening the `/conda-store` in a new browser tab.